docs/source/markdown/podman-run.1.md

```diff
@@ -901,17 +901,19 @@ Ulimit options. You can use **host** to copy the current configuration from the
 
 Sets the username or UID used and optionally the groupname or GID for the specified command.
 
-Without this argument the command will be run as root in the container.
+Without this argument the command will be run as root in the container, unless a USER has been
+set in the Containerfile.  If the USER was set in the Containerfile, it will be used to process
+commands within the container.
 
 **--userns**=**auto**|**host**|**keep-id**|**container:**_id_|**ns:**_namespace_
 
-Set the user namespace mode for the container.  It defaults to the **PODMAN_USERNS** environment variable.  An empty value means user namespaces are disabled.
+Set the user namespace mode for the container.  It defaults to the **PODMAN_USERNS** environment variable.  An empty value ("") means user namespaces are disabled.
 
 - **auto**: automatically create a namespace.  It is possible to specify other options to `auto`.  The supported options are
   **size=SIZE** to specify an explicit size for the automatic user namespace.  e.g. `--userns=auto:size=8192`.  If `size` is not specified, `auto` will guess a size for the user namespace.
   **uidmapping=HOST_UID:CONTAINER_UID:SIZE** to force a UID mapping to be present in the user namespace.
   **gidmapping=HOST_UID:CONTAINER_UID:SIZE** to force a GID mapping to be present in the user namespace.
-- **host**: run in the user namespace of the caller. This is the default if no user namespace options are set. The processes running in the container will have the same privileges on the host as any other process launched by the calling user.
+- **host**: run in the user namespace of the caller. The processes running in the container will have the same privileges on the host as any other process launched by the calling user.
```

Comment:
This is the default, not private

Reply:
How about running as root vs non-root? Is this default the same for non-root too? I thought that running as non-root creates a new userns.

Reply:
This is difficult to explain.

Root inside of the rootless container is your UID, so theoretically it has the same privs as a process run by your user.
Bottom line all containers will run within the same user namespace, and you will gain no separation advantage between containers launched by the same user.  All of the processes in containers will run within the same range of UIDs that other containers you launch run with.

```
$ podman run --userns=host alpine cat /proc/self/uid_map
         0       3267          1
         1     100000      65536
$ podman run --userns=host alpine cat /proc/self/uid_map
         0       3267          1
         1     100000      65536
```

Reply:
> This is the default, not private

Thanks, I was looking at the wrong field with a default "Private" value (userns mode) when I change this.  I've changed the default to `host`.